earth_enterprise/src/third_party/apache2/SConscript

```diff
@@ -26,8 +26,9 @@ apache_patches = ['apache-ge-enablemanual.patch',
                   'apache-ge-htdocsallowoverride.patch',
                   'apache-ge-httpd.conf.in-cgi-bin_admin.patch',
                   'apache-ge-httpd.conf.in-handle_http_errors.patch',
-                  'apache-ge-httpd-ssl.conf.in.patch',
-                  'apache-ge-httpd-ssl.conf.in-include_location_ssl.patch']
+                  'apache-ge-httpd-ssl.conf.in-include_location_ssl.patch',
+                  'apache-gee-httpd-ssl.conf.in.patch'
```

Comment:
just wanted to keep the new ones added after open gee separate.  probably does not matter after we review the existing patches.